### PR TITLE
Update linter to remove bug

### PR DESCRIPTION
**Pull request by seanpzk**
- Remove standard js for typescript and javascript
- Standardize to use ESLint for both languages
- Update README to give a summary of the linters used

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 # CS3219 Project (PeerPrep) - AY2425S1
-## Group: Gxx
 
-### Note: 
-- You can choose to develop individual microservices within separate folders within this repository **OR** use individual repositories (all public) for each microservice. 
-- In the latter scenario, you should enable sub-modules on this GitHub classroom repository to manage the development/deployment **AND** add your mentor to the individual repositories as a collaborator. 
-- The teaching team should be given access to the repositories as we may require viewing the history of the repository in case of any disputes or disagreements. 
+## Group: G08
+
+## CI/CD Pipeline
```
Comment (Pughal77) [inline README.md:7]: Nice that we are documenting this in the readme so that it would be easier to write our justifications for the project report!

**Review by Pughal77 — COMMENTED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (DonovanJJ) [reply README.md:7]: Yeap! @seanpzk can probably start documenting these for the final report as well jic forget

**Review by DonovanJJ — APPROVED** [review]
Looks good

**Review by Pughal77 — APPROVED** [review]
LGTM!

**Pull request merged**